### ISSUE TITLE
Scalatra 2.5.1 breaks CORS-support for Swagger

### ISSUE THREAD

**Issue by sigand** [scalatra/scalatra#695]
Upgrading from Scalatra 2.5.0 to 2.5.1 breaks the CORS needed for Swagger in http://petstore.swagger.io/

Response header Access-Control-Allow-Origin is no longer returned. This is tested with default settings

**Comment by nilskp** [comment]
This is not isolated to Swagger, it's a general CORS problem. We have to roll back.

**Comment by takezoe** [comment]
@sigand @nilskp 

Sorry for my slow reply.

In Scalatra 2.5.1, a security issue in CORSSupport has been fixed in #635. As the result, the origin which is sent from a client must be declared in the init param `org.scalatra.cors.allowedOrigins` like:
	
```scala
context.initParameters("org.scalatra.cors.allowedOrigins") 
  = "http://example.com:8080,http://foo.example.com"
```

Since the default of this init param is `"*"`. So if you don't specify this parameter, I guess it works.

What is your configuration?

**Comment by takezoe** [comment]
I confirmed that CORS support works with the following configuration:
```scala
context.initParameters("org.scalatra.cors.allowedOrigins") = "http://petstore.swagger.io"
```
I close this issue once, but feel free to reopen if you still have a trouble in CORS support.
